schema/annotation-source.yaml

```diff
@@ -161,30 +151,51 @@ $defs:
       A Statement (aka ‘Assertion’) represents a claim of purported truth as made by a particular agent, 
       on a particular occasion.
     heritable_properties:
+      target_proposition:
+        oneOf:
+        - $ref: "#/$defs/Proposition"
+        - $ref_curie: gks.core:CURIE
+        description: The Proposition about which the Statement is made.
+      evidence_direction:
+         type: string
+         enum: [ "supports", "disputes", inconclusive" ] 
```

Comment:
> "disputes"

FWIW I think the term 'disputes' is more fitting to describe evidence direction when the evidence does not support the target proposition , compared to "opposes".  To me it's more natural to say 'this evidence disputes your claim', than 'this evidence opposes your claim').  But I defer to you others here. 

> "inconclusive"

Similarly, since we have a field here specific to evidence direction, I think it more natural to say "the evidence is inconclusive", rather than "the evidence is uncertain". When it comes to evidence strength, a curator can be certain that the evidence is inconclusive.